web/src/user_group_pill.ts

```diff
@@ -1,10 +1,29 @@
 import * as user_groups from "./user_groups";
 
-function display_pill(group) {
-    return group.name + ": " + group.members.size + " users";
+type UserGroup = {
+    id: number;
+    name: string;
+    description: string;
+    members: Set<number>;
+    is_system_group: boolean;
+    direct_subgroup_ids: Set<number>;
+};
+
+function display_pill(group: UserGroup): string {
+    return `${group.name}: ${group.members.size} users`;
+}
+
+interface CurrentItem {
```

Comment:
How did you pick this name? I think something like `PillItem` would probably be more appropriate if this is a shared data structure with other pill modules.

Reply:
Thanks for the review, I have updated the PR accordingly